force-app/main/default/classes/SOQL.cls

```diff
@@ -201,7 +274,7 @@ public with sharing class SOQL {
      * @return A list of Cases
      */
     public static List<Case> getCasesWithStatus() {
-        return null; // Replace null with the variable you used to store the result
+        return [SELECT Id, CaseNumber, CreatedDate FROM Case WHERE Status = 'Open' ORDER BY CreatedDate ASC LIMIT 10 OFFSET 5]; // Replace null with the variable you used to store the result
```

Comment:
Try using not closed instead of Open since some other statuses like new can be considered open